docs/upgrade-to-1.0.md

```diff
@@ -0,0 +1,85 @@
+# Upgrading to Velero 1.0
+
+## Prerequisites
+- Velero v0.11 installed. If you're not already on v0.11, see the [instructions for upgrading to v0.11][0]. **Upgrading directly from v0.10.x or earlier to v1.0 is not supported!**
+- (Optional, but strongly recommended) Create a full copy of the object storage bucket(s) Velero is using. Part 1 of the upgrade procedure will modify the contents of the bucket, so we recommend creating a backup copy of it prior to upgrading.
+
+## Instructions
+
+### Part 1 - Rewrite Legacy Metadata
+
+#### Overview
+
+If you have any backups that were originally taken with a version prior to v0.11 (i.e. when the project was named Ark), you need to replace the legacy backup metadata in object storage with updated versions. While Velero v0.11 is backwards-compatible with these files, Velero v1.0 is not.
```

Comment:
`s/If you have any backups/In case` you have any backups.

For some reason this sounds more like a prompt to be cautious and run this part 1.

Reply:
that edit sounds slightly strange to my ear, but i pushed an update that i think accomplishes the same thing.

Reply:
Yes, looks great now.